### PR TITLE
Release preparations.

### PR DESCRIPTION
**Pull request by hernando**

### CODE REVIEW

File: doc/Changelog.md [hunk 1]
```diff
@@ -1,25 +1,27 @@
 Changelog {#Changelog}
 =========
 
-## git master {master)
+<!-- Uncomment at first change # git master {#master) -->
+
+# Release 1.6.0 (2015-10-04) {#Release010600}
```
Comment (bilgili) [inline doc/Changelog.md]: Date is wrong and I pick the 9th of November as the release date.

File: doc/Changelog.md [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
 
 * [#24](https://github.com/BlueBrain/Brion/pull/24):
```
Comment (bilgili) [inline doc/Changelog.md]: Remove the hash tags

File: README.md [hunk 3]
```diff
@@ -52,6 +47,11 @@ environments are tested:
 
 * Linux: Ubuntu 14.04 and RHEL 6 (Makefile, i386, x64)
 
-The [latest API documentation]
-(http://bluebrain.github.io/Brion-1.5/index.html) can be found on
-[bluebrain.github.io](http://bluebrain.github.io).
+Building from source is as simple as:
+
+    git clone https://github.com/BlueBrain/Brion.git
+    cd Brion
```
Comment (eile) [inline README.md]: rm

File: doc/Changelog.md (hunk 1 above)
Comment (eile) [inline doc/Changelog.md]: I picked the current date too, although this is the third way of formatting the date...

In the early projects I had 02-11-2015, in later ones 03-Nov-2015 to not confuse the Americans. I would vote for the latter format.

File: doc/Changelog.md [hunk 4]
```diff
@@ -1,25 +1,27 @@
...
-  Add brion::Target::parse() to resolve a given target name
-* [#12](https://github.com/BlueBrain/Brion/pull/12):
-  Extended brion::Synapse to also support non-merged synapse files
-* [#22](https://github.com/BlueBrain/Brion/pull/22):
-  SpikeReport DSO plugins in the LD_LIBRARY_PATH are loaded automatically.
-* [#22](https://github.com/BlueBrain/Brion/pull/22):
-  spikeConverter can process spikes to and from stream-type SpikeReport plugins.
 
-## Release 1.5.0 (2015-07-07) {changelog_1_5_0}
+# Release 1.5.0 (2015-07-07) {#Release010500}
```
Comment (eile) [inline doc/Changelog.md]: We don't need the {#targets}, they just make the github rendering ugly and add little in doxygen.

File: doc/Changelog.md [hunk 5]
```diff
@@ -55,7 +57,7 @@ Changelog {#Changelog}
 * Spike loading has been optimized (in a 12-core GPU the expected
   speed-up is above 4x for large files).
 
-## Known Bugs {Bugs}
+# Known Bugs {Bugs}
```
Comment (eile) [inline doc/Changelog.md]: rm section - does not belong here.

**Comment by hernando** [comment]
Any more comments?

**Comment by bilgili** [comment]
+1 for me

**Pull request merged**